cmd/kured/main.go

```diff
@@ -421,22 +425,23 @@ func (kb KubernetesBlockingChecker) isBlocked() bool {
 	return false
 }
 
-func rebootBlocked(blockers ...RebootBlocker) bool {
+func rebootBlocked(ctx context.Context, blockers ...RebootBlocker) bool {
+	// TODO: can we do that concurrently
```

Comment:
So that we don't commit a `// TODO` comment, could you add an issue describing this optimization instead? (agree that this is out of scope for this PR)

thank you!